panel/models/json.ts

```diff
@@ -15,7 +15,7 @@ export class JSONView extends PanelMarkupView {
 
   render(): void {
     super.render();
-    const text = this.model.text.replace(/(\r\n|\n|\r)/gm, "").replace("'", '"')
```

Comment:
Is there much harm in leaving this? Would be good to keep as much functionality in pure JS as possible.

Reply:
I have added back `.replace(/(\r\n|\n|\r)/gm, "")`. If I add back `.replace("'", '"')` it breaks the examples 1, 2, 3 and 6.

Reply:
👍